common/cards/advent-of-tcg/single-use/redstone-torch.ts

```diff
@@ -82,7 +82,7 @@ const RedstoneTorch: SingleUse = {
 							target: opponentPlayer.activeRowEntity,
 							type: 'effect',
 							log: (values) =>
-								`${values.player} detonated their ${tntCardEntities.length} $e${TNT.name}$ with $e${RedstoneTorch.name}$ to attack ${values.target} for ${values.damage} damage `,
+								`${values.player} detonated their ${tntCardEntities.length} $e${TNT.name}$ with $e${RedstoneTorch.name}$ to attack ${values.target} for ${values.health} ${values.healing ? 'healing' : 'damage'} `,
```

Comment:
I don't think this check should be necessary since TNT should never heal. I think the same for the rest of these checks.